data/campaigns/The_South_Guard/scenarios/02_Proven_by_the_Sword.cfg

```diff
@@ -354,19 +354,19 @@
         [message]
             speaker=Deoran
             image=portraits/deoran-mad.png
-            message= _ "More bandits?! They’ll feel the deadly bite of my lance!"
+            message= _ "More bandits?! Prepare to be slain by my lance!"
         [/message]
         [message]
             speaker=Minister Hylas
-            message= _ "Peace! For I am no enemy. I am Minister Hylas, advisor to Sir Loris! You came just in time! We couldn’t have held them off another day..."
+            message= _ "Peace, I am no enemy! My name is Hylas, advisor to Sir Loris and member of the town council! Thanks to your timely arrival, the city is saved!"
         [/message]
         [message]
             speaker=Deoran
-            message= _ "Indeed, our arrival was timely. But where is Sir Loris? Where is the garrison of the city? I was sent by Haldric to relieve him and take command of the South Guard, for the King has heard no news from Westin in two months."
+            message= _ "Indeed, we have defeated most of the bandits in the city. But where is Sir Loris? And where is the city garrison?"
         [/message]
         [message]
             speaker=Minister Hylas
-            message= _ "Alas, Sir Loris will find no relief now. He is buried northwest of the city where he fell. For weeks we warned him of the new power of these bandits. And yet he would never lead his men against them until their strength had waxed and ours waned."
+            message= _ "Alas, Sir Loris fell several days ago. For weeks, we warned him of the new power of these bandits, and yet he would never lead his men against them until their strength had become too great for us to handle. I had sent a messenger to the king about our plight, but some misfortune must have befallen him, for you are the first reinforcements to arrive here. If only Sir Loris had sent a rider earlier, he might have survived long enough to stand with us here."
```

Comment:
Good, but I think "fell in battle" is called for now that "buried northwest of the city" is no longer included.